test/unit/test-kernel.cpp

```diff
@@ -2634,4 +2634,56 @@ CUDA_TEST(Kernel, CudaExec_1threadexec)
   ASSERT_EQ(result, N * N * N * N);
 }
 
+
+
+CUDA_TEST(Kernel, CudaExec_fixedspillexec)
+{
+  using namespace RAJA;
+
+
+  constexpr long N = (long)2048;
```

Comment:
Would it be better to use named const variables for these constants rather than sprikling magic numbers around in the code?